codespell_lib/data/dictionary.txt

```diff
@@ -4767,52 +4914,94 @@ repulican->republican
 repulicans->republicans
 repulics->republics
 reqest->request
+reqeust->request
+reqeusts->requests
+requiere->require
+requiered->required
 requierement->requirement
 requierements->requirements
+requieres->requires
 requiering->requiring
 requireing->requiring
 requirment->requirement
+requirments->requirements
 requred->required
+requrest->request
+requrested->requested
+requresting->requesting
+requrests->requests
+requried->required
 requrirement->requirement
 requrirements->requirements
+requst->request
+requsted->requested
+requsting->requesting
+requsts->requests
 resaurant->restaurant
 resembelance->resemblance
 resembes->resembles
 resemblence->resemblance
+reserverd->reserved
+reseted->reset
 reseting->resetting
+resetted->reset
 resevoir->reservoir
+resgister->register
+resgisters->registers
 residental->residential
 resignement->resignment
 resistable->resistible
 resistence->resistance
 resistent->resistant
+resizeable->resizable
```

Comment:
resizeable is valid http://www.yourdictionary.com/resizeable

Reply:
please remove then

Reply:
Done. I changed  
`resizeable->resizable`  
to  
`resizeble->resizable`